apps/glusterfs/app.go

```diff
@@ -97,7 +97,7 @@ func NewApp(configIo io.Reader) *App {
 	// Setup BoltDB database
 	app.db, err = bolt.Open(dbfilename, 0600, &bolt.Options{Timeout: 3 * time.Second})
 	if err != nil {
-		logger.Warning("Unable to open database.  Retrying using read only mode")
+		logger.Warning("Unable to open database, Err: %v. Retrying using read only mode", err)
```

Comment:
Generally good, but this `"Unable to open database, Err: %v."` reads a little awkward.
I find that usually s/th like this is sufficient: `"Unable to open database: %v."

For readability of the logs, we might also consider doing a new message / line for the `"Retrying using read only mode"` message.